README.md

```diff
@@ -1,3 +1,80 @@
-# orcas-phantom-back
 [![Build Status](https://travis-ci.org/atlp-rwanda/orcas-phantom-backend.svg?branch=ch-traviscov-backend)](https://travis-ci.org/atlp-rwanda/orcas-phantom-backend)
 [![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/orcas-phantom-backend/badge.svg?branch=ch-traviscov-backend)](https://coveralls.io/github/atlp-rwanda/orcas-phantom-backend?branch=ch-traviscov-backend)
+
+# orcas-phantom-backend
+
+Phantom is a web application that allows passengers to track bus location and movements around Kigali.
+
+# API Host :globe_with_meridians:
+
+API endpoints are hosted [Here](https://phantom-backend.herokuapp.com/)
+
+# API Documentation :pencil:
+API endpoints Documentation are hosted [Here](https://phantom-backend.herokuapp.com//swaggerDocument/)
+
+## Technologies && Tools
+
+* [NodeJS](https://nodejs.org/) - JavaScript Runtime Environment
+* [ExpressJs](https://expressjs.com/) - A Minimal  Web Application Framework
+* [postgres](https://www.postgresql.org/) -open source object-relational database system.
+* [jest](https://jestjs.io/) - A JavaScript test framework for Node.js programs, asynchronous testing, test coverage reports, and use of any assertion library
```

Comment:
Thank you @giranezafiacre but here in tools and technology I think you can add other more:

- Docker
- Swagger documentation
- ESLint
- CI/CD Method used in the project

Reply:
I thought it will be good to avoid to put a lot development technologies @Isaackomeza, what others think @Malaba6 and @fistonhn?